server/config/subject_page.proto

```diff
@@ -44,6 +44,8 @@ message EventTypeSpec {
   string color = 4;
   // Default severity filter to use for this event type.
   SeverityFilter default_severity_filter = 5;
+  // Severity filter overrides, keyed by place type.
+  map<string, SeverityFilter> place_type_severity_filter = 10;
```

Comment:
I wonder if instead of adding this to the proto, we could just have a map somewhere for this .. because this proto is what is getting sent to the client side and it doesn't look like we're actually sending this information?

Reply:
we used to have that override in code and i feel it's messy and easy to forget. this way all the definitions related to it is in one place and easy to update.

Reply:
hm, ok, that makes sense. Maybe add a comment that this is just used to override default_severity_filter and that it's not actually sent to the client